Paco-Server/src/com/google/sampling/experiential/server/EventJsonUploadProcessor.java

```diff
@@ -103,8 +113,20 @@ private String toJson(List<Outcome> outcomes) {
     }
     return results;
   }
-
-  private Outcome postEvent(JSONObject eventJson, int eventId, String who, String appIdHeader, String pacoVersionHeader) throws Throwable {
+  
+//  public static <T> T getGenType(JSONObject json, String attribute) throws JSONException{
```

Comment:
Well, I tried writing this generic method to retrieve the different json attributes. This will be good for retrieving all string values, but for long and other data types, we still need to do the conversion etc in addition to this code. So, removing it for now.